Restfull API/api/instance/config.py

```diff
@@ -0,0 +1 @@
+SECRET_KEY = 'p9Bv<3Eid9%$i01'
```

Comment:
You can move this into the Config class that holds the common configurations for the app.